test/e2e/testnet/node.go

```diff
@@ -28,21 +29,25 @@ import (
 )
 
 const (
-	rpcPort        = 26657
-	p2pPort        = 26656
-	grpcPort       = 9090
-	prometheusPort = 26660
-	tracingPort    = 26661
-	dockerSrcURL   = "ghcr.io/01builders/celestia-app" // TODO: revert to celestiaorg
-	secp256k1Type  = "secp256k1"
-	ed25519Type    = "ed25519"
-	remoteRootDir  = "/home/celestia/.celestia-app"
-	txsimRootDir   = "/home/celestia"
+	rpcPort               = 26657
+	p2pPort               = 26656
+	grpcPort              = 9090
+	listenAddressPort     = 26658
+	grpcListenPort        = 9099
```

Comment:
[question] is there a rationale behind 9099? I wonder if I should update https://github.com/celestiaorg/celestia-app/pull/4637 to use 9099 instead of 9098

Reply:
no this was just an arbitrary port that also needs to be exposed separate from the 9090 gRPC server.